kahuna/public/js/edits/metadataDiff.js

```diff
@@ -0,0 +1,26 @@
+export const getMetadataDiff = (image, metadata) => {
+  const diff = {};
+  const originalMetadata = image.data.originalMetadata;
+  // jscs has a maximumLineLength of 100 characters, hence the line break
+  var keys = new Set(
+    Object.keys(metadata).concat(Object.keys(originalMetadata))
+  );
+
+  keys.forEach((key) => {
+    if (originalMetadata[key] === metadata[key]) {
```

Comment:
This logic is expressed much more cleanly here – as far as I can see it's like for like 🤞